vendor/mozjexl.jsm

```diff
@@ -763,16 +776,47 @@ exports.FilterExpression = function(ast) {
  * @private
  */
 exports.Identifier = function(ast) {
+  function contextHasProp(context, prop) {
+    // If this is primitive object (eg "string") we can't check
+    // this, which is why we have to be careful not to look in things
+    // that aren't objects.
+    if (typeof context == "object" && !(prop in context)) {
+      return false;
+    }
+
+    return true;
+  }
+
+  let that = this;
   if (ast.from) {
     return this.eval(ast.from).then(function(context) {
```

Comment:
If you change this to an arrow function, it will bind `this` correctly and you don't have to do the `that` vs `this` juggling

Reply:
mozjexl.jsm is just the built artifact, so it'll get updated when I fix the other instance.